doc/lsst.ap.verify/command-line-reference.rst

```diff
@@ -0,0 +1,139 @@
+.. _ap-verify-cmd:
+
+######################
+Command Line Reference
+######################
+
+This page describes the command-line arguments and environment variables used by ``ap_verify``.
```

Comment:
You might want to distinguish between `ap_verify` the EUPS package and `ap_verify` the command-line executable.

One way might be to have the EUPS package reference link to the package homepage.

Then the command-line references might be called `ap_verify.py` rather than `ap_verify` and also link to this reference page.

Reply:
I'm not sure I understand. What exactly is the distinction (I think I almost always mean "piece of software" when I say "ap_verify"), and why would I want `ap_verify`'s documentation to link to itself?

Reply:
I say this because ``ap_verify`` the EUPS packages doesn't have command-line arguments, but ``ap_verify.py`` the command-line script does have arguments.